pdns/dnsdistdist/dnsdist-healthchecks.cc

```diff
@@ -151,6 +158,7 @@ class HealthCheckQuerySender : public TCPQuerySender
 
   void notifyIOError(InternalQueryState&& query, const struct timeval& now) override
   {
+    ++d_data->d_ds->d_healthCheckMetrics.d_networkErrors;
```

Comment:
Several cases of `d_networkErrors` are lumped together. Does it make sense to separate those?

Reply:
I don't think so, as my current plan is to get a better understanding of the generic kind of failures. If we want more details we can turn the "verbose healthchecks" option to get detailed errors in the logs.